_doc/developer/general/documentation-style-guide.md

```diff
@@ -0,0 +1,355 @@
+
+### English language conventions
+
+For the sake of consistency and uniformity, the Qubes documentation aims to follow the conventions of American English, where applicable. (Please note that this is an arbitrary convention for the sake consistency and not a value judgment about the relative merits of British versus American English.)
+
+## Organizational guidelines
+
+### Do not duplicate documentation
+
+Duplicating documentation is almost always a bad idea. There are many reasons for this. The main one is that almost all documentation has to be updated as some point. When similar documentation appears in more than one place, it is very easy for it to get updated in one place but not the others (perhaps because the person updating it doesn't realize it's in more than once place). When this happens, the documentation as a whole is now inconsistent, and the outdated documentation becomes a trap, especially for novice users. Such traps are often more harmful than if the documentation never existed in the first place. The solution is to **link** to existing documentation rather than duplicating it. There are some exceptions to this policy (e.g., information that is certain not to change for a very long time), but they are rare.
+
+### Core vs. external documentation
+
+Core documentation resides in the [Qubes OS Project's official repositories](https://github.com/QubesOS/), mainly in [qubes-doc](https://github.com/QubesOS/qubes-doc). External documentation can be anywhere else (such as forums, community websites, and blogs), but there is an especially large collection in the [Qubes Forum](https://forum.qubes-os.org/docs). External documentation should not be submitted to [qubes-doc](https://github.com/QubesOS/qubes-doc). If you've written a piece of documentation that is not appropriate for [qubes-doc](https://github.com/QubesOS/qubes-doc), we encourage you to submit it to the [Qubes Forum](https://forum.qubes-os.org/docs) instead. However, *linking* to external documentation from [qubes-doc](https://github.com/QubesOS/qubes-doc) is perfectly fine. Indeed, the maintainers of the [Qubes Forum](https://forum.qubes-os.org/) should regularly submit PRs against the documentation index (see [How to edit the documentation index](https://doc.qubes-os.org/en/latest/developer/general/how-to-edit-the-documentation.html#how-to-edit-the-documentation-index)) to add and update Qubes Forum links in the ["External documentation"](https://qubes-doc-rst.readthedocs.io/en/latest/index.html#external-documentation) section of the documentation table of contents.
```

Comment:
Same thing here:
https://doc.qubes-os.org/en/latest/developer/general/how-to-edit-the-documentation.html#how-to-edit-the-documentation-index
redirects to
https://www.qubes-os.org/doc/how-to-edit-the-documentation/
missing `#how-to-edit-the-documentation-index`.

Either link to
https://www.qubes-os.org/doc/how-to-edit-the-documentation/#how-to-edit-the-documentation-index
directly or use magic via
`/doc/how-to-edit-the-documentation/#how-to-edit-the-documentation-index`

Not tested.